### ISSUE TITLE
Legends

### ISSUE THREAD

**Issue by QueenKraken** [elifesciences/enhanced-preprints-server#466]
Description
As an eLife reader I would like reviewed pre prints figures to have a corresponding legend to be accurately rendered and polished to demonstrate professional standards

**Comment by discodavey** [comment]
Are the figures still going to be images or will that be changing and how would this work with legends being introduced?

**Comment by discodavey** [comment]
Need more information and screenshots.

**Comment by QueenKraken** [comment]
@discodavey Yes the figures are still going to be images for the foreseeable. We need to discover the solution.
